### PR TITLE
[Wallet] Various fixes in prep for staging release

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

* Add .env.staging to use alfajoresstaging
* Use button instead of text for copy button on backup screen (see screenshot)
* Don't write temp wallet address to logs
* Skip escrow withdrawal if verification fails
* Minor style improvement for import contacts screen
* Ignore ruby artifacts while linting and compiling

### Tested

Ran through onboarding flow and key send/exchange flows for alfajoresstaging

### Backwards compatibility

Compatible

improved-copy-button

### CODE REVIEW

File: packages/blockchain-api/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Build and start:
 Deploy your app. The project will be built automatically by Google Cloud Build:
 
     using celotool:
-        yarn run cli deploy upgrade blockchain-api --config CONFIG_FILE --verbose --celo-env CELO_ENV
+        celotool deploy upgrade blockchain-api --config CONFIG_FILE --verbose --celo-env CELO_ENV
```
Comment (martinvol) [inline packages/blockchain-api/README.md:27]: Docs refers to it as `celotooljs` https://github.com/celo-org/celo-monorepo/blame/1a6a140dbba4c88a0ef25f752e4ba09e83bc1978/packages/celotool/README.md#L18

File: packages/mobile/.env.alfajoresstaging [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ENVIRONMENT=alfajoresstaging
+DEFAULT_TESTNET=alfajoresstaging
+BLOCKCHAIN_API_URL=https://alfajoresstaging-dot-celo-testnet.appspot.com/
+DEV_SETTINGS_ACTIVE_INITIALLY=true
+FIREBASE_ENABLED=true
+SECRETS_KEY=integration
```
Comment (martinvol) [inline packages/mobile/.env.alfajoresstaging]: this should be production, but now that I see the shipped app to the play store uses integration as well. At some point we should change that.

File: packages/mobile/src/invite/saga.ts [hunk 3]
```diff
@@ -213,7 +213,7 @@ export function* redeemInviteSaga(action: RedeemInviteAction) {
         throw e
       }
     }
-    Logger.debug(TAG + '@redeemInviteCode', 'Added temp account to wallet: ' + tempAccount)
```
Comment (martinvol) [inline packages/mobile/src/invite/saga.ts:216]: I've found this string useful in the past., it's just the public key, right?

File: packages/mobile/src/redux/sagas.ts [hunk 4]
```diff
@@ -27,6 +27,7 @@ const loggerBlacklist = [
   'WEB3/SET_COMMENT_KEY',
   'IDENTITY/UPDATE_E164_PHONE_NUMBER_ADDRESSES',
   'IDENTITY/FETCH_PHONE_ADDRESSES',
+  'INVITE/REDEEM_INVITE',
```
Comment (martinvol) [inline packages/mobile/src/redux/sagas.ts:30]: Nice catch

File: packages/blockchain-api/README.md (hunk 1 above)
Comment (jmrossy) [reply packages/blockchain-api/README.md:27]: We should change the docs to make it just celotool now that celotooljs is the only celotool

Comment (jmrossy) [reply packages/blockchain-api/README.md:27]: I will update it in a follow-up PR

File: packages/mobile/src/invite/saga.ts (hunk 3 above)
Comment (jmrossy) [reply packages/mobile/src/invite/saga.ts:216]: Okay, will restore for now

**Review by jeanregisser — DISMISSED** [review]
Good job 👍 :shipit:

**Review by martinvol — APPROVED** [review]

**Pull request merged**